jasmine/spec/string-extension-tests.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+describe('String Extension', function() {
+
+  describe('Has Vowels', function() {
+    it('should return true if the string contains vowels', function() {
+      expect('hi'.hasVowels()).toBeTruthy();
+      expect('HI'.hasVowels()).toBeTruthy();
+    });
+  });
+
+  describe('To Upper', function() {
+    it('should return the string passed as all upper cases characters.', function() {
+      expect('hi'.toUpper()).toEqual('HI');
+      expect('HI'.toUpper()).toEqual('HI');
+    });
+  });
+
+  describe('To Lower', function() {
+    it('should return the string passed as all lower cases characters.', function() {
```

Comment:
I incorporate that I think you are commenting on an outdated copy

<expect('?'.toUpper()).toEqual('?');>